abc.templates/README.md

```diff
@@ -0,0 +1,17 @@
+# Guardian
+
+### Installation with abc CLI
+
+The following command installs Guardian workflows in the current directory.
+
+```shell
+abc templates render \
+  -input=guardian_wif_provider=<WIF_PROVIDER> \
+  -input=guardian_service_account=<SERVICE_ACCOUNT> \
+  -input=guardian_state_bucket=<GUARDIAN_STATE_BUCKET> \
+  github.com/abcxyz/guardian.git//abc.templates/workflows
+```
+
+#### Optional inputs:
+- `terraform_version`: Defaults to `1.5.6`.
```

Comment:
I believe we can only currently support 1.5.4 because of token minter and its version checksums: https://github.com/abcxyz/secure-setup-terraform/blob/v0.2.7/terraform-checksums.json